add.go

```diff
@@ -0,0 +1,104 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/bndw/pick/errors"
+	"github.com/bndw/pick/utils"
+)
+
+const (
+	passwordLength = 25
+)
+
+func addCommand(args ...string) int {
+	safe, err := loadSafe()
+	if err != nil {
+		return handleError(err)
+	}
+
+	name, username, password, errCode := parseAddArgs(args)
+	if errCode > 0 {
+		return errCode
+	}
+
+	err = safe.Add(name, username, password)
+	if _, conflict := err.(*errors.AccountExists); conflict {
+		if overwrite(name) {
+			// Remove the existing credential
+			rerr := safe.Remove(name)
+			if rerr != nil {
+				return handleError(rerr)
+			}
+
+			addCommand([]string{name, username, password}...)
+		}
+	} else if err != nil {
+		return handleError(err)
+	}
+
+	fmt.Println("Credential added")
```

Comment:
@leonklingele To clarify, are you recommending the `add` command prints the password of the newly added credential to stdout?